packages/@react-aria/interactions/src/utils.ts

```diff
@@ -30,6 +30,13 @@ export function isVirtualClick(event: MouseEvent | PointerEvent): boolean {
     return true;
   }
 
+  // Android TalkBack's detail value varies depending on the event listener providing the event so we have specific logic here instead
+  // If pointerType is defined, event is from a click listener. For events from mousedown listener, detail === 0 is a sufficient check
+  // to detect TalkBack virtual clicks.
+  if (isAndroid() && (event as PointerEvent).pointerType != null) {
+    return event.type === 'click' && event.buttons === 1;
```

Comment:
Are you sure this won't happen without talkback? `event.buttons` should usually be 1, I don't think that's unique?

Reply:
From my testing `event.buttons` is 0 unless triggered by Talkback double tap.

Reply:
is it possible to connect a mouse to an android device though? in that case, would it have event.buttons = 1? Maybe matching the check we do in usePress would be better? I guess we'd have to add something to `handlePointerEvent`

Reply:
From what I observed, click events always have `event.buttons = 0` on Chrome/FF/Safari. It'd be a different story if the event came from a mousedown/pointerdown listener AND was a PointerEvent (aka Chrome) but those will have `type = mousedown` and `type = pointerdown` and thus will return false in the above logic. 

I tested my bluetooth mouse with my android phone and observed the same as above, `event.buttons = 0`

Reply:
ok. I'm a little concerned that this assumption might not hold in the future, but we can merge for now.

Reply:
yeah, I agree its less than ideal and is definitely prone to breaking very easily if any of the information provided by these events change in the future. If we'd prefer to not have this in `isVirtualClick`, I can move this detection back to `useFocusVisible`'s `handleClickEvent` like I had in https://github.com/adobe/react-spectrum/pull/3266/commits/92edd2bf9fdbfde3f8e0703c7165b2665abc3e44